### DISCUSSION TITLE
The test time of the calibration network should be as short as possible

### DISCUSSION THREAD

**Discussion by YiRuiLiher** [filecoin-project/lotus#5760] [Ideas] [answered]
Now the network is extremely unfavorable for testing. It takes at least 6 hours to see the results when we test our own programs. This is very inefficient for testing. I think the cost of testing should be reduced as much as possible. Conducive to the production of excellent programs.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
i can not agree more on this. the removal of 512MB sectors basically removed the possibility to use the network for testing applications. 

@YiRuiLiher the nerpa is said to be reset soon. if they support smaller sector sizes we will move our testnet infrastructure there to have a place to test in fast paced cycles.

**Reply by YiRuiLiher** [reply]
Thank you

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
@YiRuiLiher nerpa is abck with 512MiB sectors! join the #fil-net-nerpa-discuss chan on slack!
